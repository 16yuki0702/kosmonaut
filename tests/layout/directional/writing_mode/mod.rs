use crate::util::dump_layout_cmd;
use insta::assert_snapshot;

#[test]
fn vertical_lr_block_box_only() {
    let mut dump_layout_cmd = dump_layout_cmd();
    dump_layout_cmd
        .arg("--files")
        .arg("tests/websrc/directional/writing_mode/vertical-lr-block-box-only.html")
        .arg("tests/websrc/directional/writing_mode/vertical-lr-block-box-only.css")
        .succeeds();
    assert_snapshot!(dump_layout_cmd.stdout());
}
