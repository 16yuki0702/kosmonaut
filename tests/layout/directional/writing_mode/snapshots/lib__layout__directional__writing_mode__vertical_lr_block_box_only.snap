---
source: tests/layout/directional/writing_mode/mod.rs
expression: dump_layout_cmd.stdout()
---
DOCUMENT Inline LayoutBox at (0, 0) size 1920x1216
  DOCTYPE Inline LayoutBox at (0, 0) size 1920x0
  HTML Block LayoutBox at (0, 0) size 1920x1216
     AnonymousInline LayoutBox at (0, 0) size 1920x0
      TEXT Inline LayoutBox at (0, 0) size 1920x0
    BODY Block LayoutBox at (1928, 8) size 1200x1904
       AnonymousInline LayoutBox at (0, 0) size 0x0
        TEXT Inline LayoutBox at (0, 0) size 0x0
        TEXT Inline LayoutBox at (0, 0) size 0x0
        TEXT Inline LayoutBox at (0, 0) size 0x0
        TEXT Inline LayoutBox at (0, 0) size 0x0
      DIV Block LayoutBox at (8, 8) size 400x200
      DIV Block LayoutBox at (408, 8) size 400x200
      DIV Block LayoutBox at (808, 8) size 400x200

